class.json-api-endpoints.php

```diff
@@ -1166,6 +1168,12 @@ function get_author( $author, $show_email_and_ip = false ) {
 			if ( defined( 'IS_WPCOM' ) && IS_WPCOM && ! $is_jetpack ) {
 				$active_blog = get_active_blog_for_user( $ID );
 				$site_id     = $active_blog->blog_id;
+				if ( $site_id > -1 ) {
+					$site_visible = (
+						-1 != $active_blog->public ||
+						is_private_blog_user( $site_id, get_current_user_id() )
```

Comment:
Just a note that this function is not defined in Jetpack, but it is within the `IS_WPCOM` check. Text editors will not be happy, but it won't actually cause any errors.